stable/redis-ha/README.md

```diff
@@ -93,6 +93,8 @@ The following table lists the configurable parameters of the Redis chart and the
 | `exporter.port`           | Exporter port                                                                                                                                                                                            | `9121`                                                                                     |
 | `exporter.annotations`    | Prometheus scrape annotations                                                                                                                                                                            | `{prometheus.io/path: /metrics, prometheus.io/port: "9121", prometheus.io/scrape: "true"}` |
 | `exporter.extraArgs`      | Additional args for the exporter                                                                                                                                                                         | `{}`                                                                                       |
+| `exporter.script.configmap`             | Let you run a custom lua script from a configmap. The corresponding environment variable `REDIS_EXPORTER_SCRIPT` will be set automatically                                                 |                                                                                            |
```

Comment:
This is too verbose (although helpful): 

```suggestion
| `exporter.script.configmap`             | Run custom Lua script as ConfigMap. Sets env var `REDIS_EXPORTER_SCRIPT`.                                                 |                                                                                            |
```

Reply:
Thanks for the suggestion, I had just copied the description exactly from the prometheus-redis-exporter chart README.